DesignSafe-CI/portal
# Task/DES-3065: Use cloud.data system for HPC work directories

## Overview: ##
Replace the tg-owned HPC works system with cloud.data and users' personal home directories
## PR Status: ##


## Related Jira tickets: ##

* [DES-3065](https://tacc-main.atlassian.net/browse/DES-3065)

## Summary of Changes: ##
- Replace `designsafe.storage.frontera.work` with user-owned HPC work directories
- Update the user profile model to pull and cache their homedir from TAS so it is available in the frontend.

## Testing Steps: ##
1. Click on "HPC Work" in the data files sidebar and check that your personal $WORK directory is listed.
2. Copy a file into your Work directory from My Data.

## UI Photos:

## Notes: ##
- Does not make cloud.data selectable for file inputs. More work needs to be done to add notes/metadata to the system.
